f600233d1b154d507b782f84604bcb12/update/sys_script_include_324825d997a7d5106fa8b4b3f153af7c.xml

```diff
@@ -44,15 +44,34 @@ RequirementsChecklist.prototype = {
       dodaac: contact.dodaac.toString()
     };
   },
+  /**
+	 * Gets a list of Contracts related to an acquisition package.
+	 * @return {array} a list of contracts
+	*/
+  //TODO refactor this into ATATUtil
+	getContractInformation: function(packageId) {
 
+		try{
+      var table = "x_g_dis_atat_current_contract_and_recurring_information"
+			var contracts = RequirementsChecklist.prototype.atatUtil.findPackageRecords(table, packageId)
+      var fields = ["contract_number", "current_contract_exists", "contract_order_expiration_date", 
+      "incumbent_contractor_name", "task_delivery_order_number"] 
 
+			return RequirementsChecklist.prototype.atatUtil.snakeToCamelCase(contracts, fields);
+		} catch (error) {
+			throw this.errUtil.createError(
+				"Dow --> getContractInformation(): " + error,
```

Comment:
```suggestion
				"RequirementsChecklist --> getContractInformation(): " + error,
```

Reply:
Fixed in https://github.com/dod-ccpo/atat-snow/pull/379/commits/469ff0ab0773235b1573c1cda94cd42158ad1602